controller/README.md

```diff
@@ -1,33 +1,39 @@
-# Source: https://www.learnpython.org/en/Hello%2C_World%21
```

Comment:
Please keep all of the `Hello World` code.  It will verify that developers have Python installed.

Reply:
Got it, I'll restore it. Is this README the appropriate place for me to add my documentation? Or should it just be what was there from before this PR?

Reply:
This `README.md`